### DISCUSSION TITLE
Config the queueSize for different queue

### DISCUSSION THREAD

**Discussion by Mr-Milk** [nextflow-io/nextflow#2258] [Q&A]
Hi, Our school's HPC has different nodes that have different user limitations. Some of the nodes (`fhs-fast`) allow me to run 3 jobs at the same time and some (`highmem`) allow me to run 5 jobs. But when I config it in this way
```
process {
    executor = "slurm"
    queue = "fhs-fast,highmem,serial-normal,serial-short,"
}

executor {
    queueSize = 10
}
```
Nextflow will try to submit all the jobs to `fhs-fast` until it reaches the `queueSize` and then it will try to submit it to `highmem`, Is there any way I could set `queueSize` for different nodes or this is actually a new feature?

Thanks!

**Comment by pditommaso** [comment]
No, that's not possible
